### ISSUE TITLE
Slamdunk crashes with plain fastq files

### ISSUE THREAD

**Issue by binyaminZ** [t-neumann/slamdunk#28]
hi,
i successfully installed slamdunk, it seems to complete the map and filter dunks, however i get the following error for either running all or count dunks:

```
Running slamDunk tcount for 1 files (1 threads) Traceback (most recent call last):
  File "/apps/RH7U2/gnu/python/2.7.15/bin/slamdunk", line 11, in <module>
    load_entry_point('slamdunk==0.2.4', 'console_scripts', 'slamdunk')()
  File "/apps/RH7U2/gnu/python/2.7.15/lib/python2.7/site-packages/slamdunk/slamdunk.py", line 478, in run
    runAll(args)
  File "/apps/RH7U2/gnu/python/2.7.15/lib/python2.7/site-packages/slamdunk/slamdunk.py", line 310, in runAll
    results = Parallel(n_jobs=n, verbose=verbose)(delayed(runCount)(tid, dunkbufferIn[tid], referenceFile, args.bed, args.maxLength, args.minQual, args.strictTCs, dunkPath, snpDirectory) for tid in range(0, len(samples)))
  File "/apps/RH7U2/gnu/python/2.7.15/lib/python2.7/site-packages/joblib/parallel.py", line 779, in __call__
    while self.dispatch_one_batch(iterator):
  File "/apps/RH7U2/gnu/python/2.7.15/lib/python2.7/site-packages/joblib/parallel.py", line 625, in dispatch_one_batch
    self._dispatch(tasks)
  File "/apps/RH7U2/gnu/python/2.7.15/lib/python2.7/site-packages/joblib/parallel.py", line 588, in _dispatch
    job = self._backend.apply_async(batch, callback=cb)
  File "/apps/RH7U2/gnu/python/2.7.15/lib/python2.7/site-packages/joblib/_parallel_backends.py", line 111, in apply_async
    result = ImmediateResult(func)
  File "/apps/RH7U2/gnu/python/2.7.15/lib/python2.7/site-packages/joblib/_parallel_backends.py", line 332, in __init__
    self.results = batch()
  File "/apps/RH7U2/gnu/python/2.7.15/lib/python2.7/site-packages/joblib/parallel.py", line 131, in __call__
    return [func(*args, **kwargs) for func, args, kwargs in self.items]
  File "/apps/RH7U2/gnu/python/2.7.15/lib/python2.7/site-packages/slamdunk/slamdunk.py", line 199, in runCount
    tcounter.computeTconversions(ref, bed, inputSNP, bam, maxLength, minQual, outputCSV, outputBedgraphPlus, outputBedgraphMinus, strictTCs, log)
  File "/apps/RH7U2/gnu/python/2.7.15/lib/python2.7/site-packages/slamdunk/dunks/tcounter.py", line 125, in computeTconversions
    slamseqInfo = SlamSeqInfo(bam)
  File "/apps/RH7U2/gnu/python/2.7.15/lib/python2.7/site-packages/slamdunk/utils/misc.py", line 67, in __init__
    DS = ast.literal_eval(getReadGroup(bam)['DS'])
KeyError: 'DS'
```

**Comment by t-neumann** [comment]
Hi,

did you run it with plain fastq-files as input or did you provide the sampleInfo.tsv file as described in the documentation?

http://t-neumann.github.io/slamdunk/docs.html#sample-file-format

**Comment by t-neumann** [comment]
Can you send me the header (samtools view -H filter.bam) of the filtered bam files you are trying to run the counting on?

**Comment by binyaminZ** [comment]
@HD     VN:1.0  SO:coordinate
@SQ     SN:chr1 LN:249250621
@SQ     SN:chr2 LN:243199373
@SQ     SN:chr3 LN:198022430
@SQ     SN:chr4 LN:191154276
@SQ     SN:chr5 LN:180915260
@SQ     SN:chr6 LN:171115067
@SQ     SN:chr7 LN:159138663
@SQ     SN:chr8 LN:146364022
@SQ     SN:chr9 LN:141213431
@SQ     SN:chr10        LN:135534747
@SQ     SN:chr11        LN:135006516
@SQ     SN:chr12        LN:133851895
@SQ     SN:chr13        LN:115169878
@SQ     SN:chr14        LN:107349540
@SQ     SN:chr15        LN:102531392
@SQ     SN:chr16        LN:90354753
@SQ     SN:chr17        LN:81195210
@SQ     SN:chr18        LN:78077248
@SQ     SN:chr19        LN:59128983
@SQ     SN:chr20        LN:63025520
@SQ     SN:chr21        LN:48129895
@SQ     SN:chr22        LN:51304566
@SQ     SN:chrX LN:155270560
@SQ     SN:chrY LN:59373566
@SQ     SN:chrM LN:16571
@RG     ID:0    SM:sample_0:NA:-1
@PG     ID:ngm  PN:ngm  VN:0.5.2        CL:" --affine 0 --argos_min_score 0 --bin_size 2 --block_multiplier 2 --bs_cutoff 6 --bs_mapping 0 --cpu_threads 1 --dualstrand 1 --fast 0 --fast_pairing 0 --force_rlength_check 0 --format 1 --gap_extend_penalty 5 --gap_read_penalty 20 --gap_ref_penalty 20 --hard_clip 0 --keep_tags 0 --kmer 13 --kmer_min 0 --kmer_skip 2 --local 1 --match_bonus 10 --match_bonus_tc 2 --match_bonus_tt 10 --max_cmrs 2147483647 --max_equal 1 --max_insert_size 1000 --max_polya 4 --max_read_length 0 --min_identity 0.650000 --min_insert_size 0 --min_mq 0 --min_residues 0.500000 --min_score 0.000000 --mismatch_penalty 15 --mode 0 --no_progress 1 --no_unal 0 --ocl_threads 1 --output SLAMDUNK2/map/Mcf7_cont_2h_4sU_S7_R1_001Aligned.sortedByCoord.out_slamdunk_mapped.sam --overwrite 1 --pair_score_cutoff 0.900000 --paired 0 --parse_all 1 --pe_delimiter / --qry Mcf7_cont_2h_4sU_S7_R1_001Aligned.sortedByCoord.out.bam --qry_count -1 --qry_start 0 --ref /home/labs/ulitsky/igoru/data/human/sequence/hg19.fa --ref_mode -1 --rg_id 0 --rg_sm sample_0:NA:-1 --sensitive 0 --silent_clip 0 --skip_mate_check 0 --skip_save 0 --slam_seq 2 --step_count 4 --strata 0 --topn 1 --trim5 12 --update_check 0 --very_fast 0 --very_sensitive 0"

**Comment by t-neumann** [comment]
Fixed in #31
